dracut/31decrypt-root/decrypt-root.service

```diff
@@ -4,14 +4,14 @@ Description=Generate and execute a systemd-cryptsetup service to decrypt the ROO
 DefaultDependencies=no
 # If systemd-networkd.service is used in the initramfs, then we should run after it, since the ROOT partition
 # might use network-bound encryption.
-After=systemd-networkd.service dev-disk-by\x2dpartlabel-ROOT.device
+After=systemd-networkd.service dev-disk-by\x2dpartlabel-ROOT.device ignition-disks.service
 Wants=dev-disk-by\x2dpartlabel-ROOT.device
-Before=dracut-initqueue.service
```

Comment:
Could you maybe update the commit message to tell why `Before=dracut-initqueue.service` and `[Install] WantedBy=ignition-subsequent.target` lines were dropped?